$OpenBSD: patch-src_miners_fs_tracker-config_c,v 1.7 2012/01/01 09:02:11 ajacoutot Exp $

Set some sane defaults (which can be changed in the preferences):
* set default indexing speed to 10 (i.e. medium value 0->20)
* don't index removable devices

--- src/miners/fs/tracker-config.c.orig	Fri Dec 16 16:24:23 2011
+++ src/miners/fs/tracker-config.c	Sat Dec 31 10:09:31 2011
@@ -37,8 +37,8 @@
 #define DEFAULT_SCHED_IDLE                       1
 #define DEFAULT_INITIAL_SLEEP                    15       /* 0->1000 */
 #define DEFAULT_ENABLE_MONITORS                  TRUE
-#define DEFAULT_THROTTLE                         0        /* 0->20 */
-#define DEFAULT_INDEX_REMOVABLE_DEVICES          TRUE
+#define DEFAULT_THROTTLE                         10        /* 0->20 */
+#define DEFAULT_INDEX_REMOVABLE_DEVICES          FALSE
 #define DEFAULT_INDEX_OPTICAL_DISCS              FALSE
 #define DEFAULT_INDEX_ON_BATTERY                 FALSE
 #define DEFAULT_INDEX_ON_BATTERY_FIRST_TIME      TRUE
